File: Data Structures & Algorithms/length-of-last-word/submission-2.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        r, length = len(s) - 1, 0
        while s[r] == ' ':
            r -= 1
        
        while r >= 0 and s[r] != ' ':
            length += 1
            r -= 1
        return length

File: Data Structures & Algorithms/climbing-stairs/submission-2.rs
impl Solution {
    pub fn climb_stairs(n: i32) -> i32 {
        let (mut one, mut two) = (1, 1);

        for _ in 0..n-1 {
            let temp = one;
            one = one + two;
            two = temp;
        }
        one
    }
}
